src/JKetelaar/Kiyoh/Models/Review.php

```diff
@@ -28,6 +28,11 @@ class Review
      */
     private $rating;
 
+    /**
+     * @var string
+     */
+    private $comments;
```

Comment:
If it is a single string, is the plural form of `comment` the right word to use? Do you have an example to clear this up for me

Reply:
Will respond from my private git account this time, as I'm not at work, so don't get confused :).

It's a comment from the company on the review.
Something like the screenshot below. ( it's the multiline text below "reactie van het bedrijf").
It's a multiline text. I agree `comments` may not be the right word to use.
Maybe `companyComment` or otherwise indeed just `comment`, as it's just a comment to the review.

![Screenshot_20191117-103607_Chrome](https://user-images.githubusercontent.com/12169252/69005725-62d8bd00-0926-11ea-83a1-717b81fe675c.jpg)

Reply:
Ahhh makes total sense, then the `comment` is the perfect naming. Thank you for elaborating. Could you add some docs to the `comment` getter, so this is also clear for future developers?